src/HotChocolate/Spatial/src/Types.Spatial/GeoJSONMultiLineStringType.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using HotChocolate;
+using HotChocolate.Types;
+using NetTopologySuite.Geometries;
+
+namespace Types.Spatial
+{
+    public class GeoJSONMultiLineStringType : ObjectType<MultiLineString>
+    {
+        protected override void Configure(IObjectTypeDescriptor<MultiLineString> descriptor)
+        {
+            descriptor.BindFieldsExplicitly();
+
+            descriptor.Implements<GeoJSONInterface>();
+
+            descriptor.Field("type").Resolver(GeoJSONGeometryType.MultiLineString);
```

Comment:
Maybe a shared resolver or resolvewith is better here.

Reply:
ok, resolvewith is still in a pull request correct so I do not have access to it in this branch and I'm not sure what a shared resolver is. Can we chat about that in slack?